README.md

```diff
@@ -109,7 +122,19 @@ However, if you'd like to daemonize it, you have to add the flag `-d`:
 docker-compose up -d
 ```
 
-Then:
+#### Optionally purge existing installation
+
+If you already had the package installed, and want a clean start:
+
+```bash
+docker-compose exec web rm composer.lock
```

Comment:
I notice this just now, but it could be written in just one line 
`docker-compose exec web rm -rf composer.lock vendor/ build/`